### ISSUE TITLE
Doesn't work with CensorTracker extension

### ISSUE THREAD

**Issue by PoorMouse** [xvzc/SpoofDPI#153]
# Describe the bug
Spoof-dpi doesn't work with [CensorTracker](https://chromewebstore.google.com/detail/censor-tracker-%E2%80%94-%D0%BE%D0%B1%D1%85%D0%BE%D0%B4-%D0%B1%D0%BB/gaidoampbkcknofoejhnhbhbhhifgdop) extension. 

I think it's because CT uses its own proxy settings.

![ff_](https://github.com/user-attachments/assets/cf65ebbc-6b0a-4c6c-b278-c2a21f25f4c0)
(Firefox settings for example)

When I disable it Youtube starts working properly.

# Client information

**OS**: macOS 10.13.6

**Version**: 0.10.6, 0.10.8

**Command to run spoof-dpi**: spoof-dpi

# Additional context

**Comment by LiquidTheDangerous** [comment]
@PoorMouse Maybe CensorTracker sets its own proxy?

**Comment by noideastowork** [comment]
Yes! Thank you for raising this topic! I also have the same issue. I really need CensorTracker for websites that block access for Russian IPs.

I can add that Adguard with DPI bypass doesn't conflict with CensorTracker. However, it seems to create a VPN configuration on macOS, but I'm not sure about that.

**Comment by LiquidTheDangerous** [comment]
@noideastowork @PoorMouse Apparently the browser is using the extension's proxy server, not the provided system proxy. Maybe if we force spoof-dpi to proxy requests through another proxy, it might work, but we need to experiment.
